07/27/2024 08:08:40 PM Logging starts
Traceback (most recent call last):
  File "./popular_projects_snippets_dataset/flask/bodies/body_21_refine_9_1.py", line 22, in <module>
    result = self.url_adapter.match(return_rule=True)  # type: ignore
AttributeError: 'Mock' object has no attribute 'url_adapter'
Command exited with non-zero status 1
1.20user 2.34system 0:00.79elapsed 449%CPU (0avgtext+0avgdata 111920maxresident)k
0inputs+24outputs (3major+25395minor)pagefaults 0swaps
